certification/internal/shell/has_prohibited_packages.go

```diff
@@ -79,23 +106,23 @@ func (p *HasNoProhibitedPackagesCheck) Help() certification.HelpText {
 // prohibitedPackageList is a list of packages commonly present in the RHEL contianer images that are not redistributable
 // without proper licensing (i.e. packages that are not under the same availability as those found in UBI).
 // TODO: Confirm these packages are the only packages in immediate scope.
-var prohibitedPackageList = []string{
-	"grub",
-	"grub2",
-	"kernel",
-	"kernel-core",
-	"kernel-debug",
-	"kernel-debug-core",
-	"kernel-debug-modules",
-	"kernel-debug-modules-extra",
-	"kernel-debug-devel",
-	"kernel-devel",
-	"kernel-doc",
-	"kernel-modules",
-	"kernel-modules-extra",
-	"kernel-tools",
-	"kernel-tools-libs",
-	"kmod-kvdo",
-	"kpatch*",
-	"linux-firmware",
+var prohibitedPackageList = map[string]struct{}{
```

Comment:
I'm not going to pretend like I have a firm grasp on language efficiency semantics, but making this a `map[string]interface{}`, and setting all values to `nil` be a more efficient? I'm understanding that you want this to be a map to facilitate lookups, which is fine, but I"m wondering if empty structs are ... less efficient.

Like I said, not even going to pretend like I fully understand the space complexity concepts as it applies in this particular case. Happy to just resolve and move forward.